vscode/microsoft-kiota/src/openApiTreeProvider.ts

```diff
@@ -238,6 +232,23 @@ export class OpenApiTreeProvider implements vscode.TreeDataProvider<OpenApiTreeN
         return undefined;
     }
 
+    private hashDescription(description: any): string {
+        const json = JSON.stringify(description);
+        return crypto.createHash('md5').update(json).digest('hex');
```

Comment:
we should probably use sha256 instead if we don't want the security folks to get mad at us (even though this is not a security feature).

Reply:
Yeah.. I figured... I used md5 because it's faster and this isn't a security feature... 
I'll return sha256